achievements/labelBabyJunior.achievement.js

```diff
@@ -0,0 +1,30 @@
+
+
+var labelBabyJunior = {
+	name: 'Label Baby Junior',
+	check: function(pullRequest, shall) {
+
```

Comment:
please remove this line too :-)

notice that all(all?) other achievements follow the same design rules, It helps to keep organized and maintainable code.
I hope that someday soon we'll have a Lint process to catch and warn on this issues.